x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/action/rolemapping/PutRoleMappingRequest.java

```diff
@@ -162,4 +162,16 @@ public void writeTo(StreamOutput out) throws IOException {
     public ExpressionRoleMapping getMapping() {
         return new ExpressionRoleMapping(name, rules, roles, roleTemplates, metadata, enabled);
     }
+
+    public static PutRoleMappingRequest fromMapping(ExpressionRoleMapping mapping) {
```

Comment:
Helper method used by the reserved handler and test to create the request from the parsed mappings.